### PR TITLE
small corrections tutorial 05

### PR DESCRIPTION
**Pull request by lcubelongren**

### CODE REVIEW

File: notes/05_infomax/1_primer_info_theory.tex [hunk 1]
```diff
@@ -446,7 +446,7 @@ \subsection{Relationship between the KL-Divergence and Mutual Information}
 D_{KL} \lbrack \, {\color{red}p_X(x) p_Y(y)} \, || \, {\color{blue}p_{X,Y}(x,y)}\, \rbrack
 \end{equation}
 
-The above is only equal if both of them are equal to zero, which is satisfied iff $p_{X,Y}(x,y) = p_X(x) p_Y(y)$.
```
Comment (kashefy) [inline notes/05_infomax/1_primer_info_theory.tex]: not a typo, iff stands for "if and only if". could make it bold

**Review by kashefy — CHANGES_REQUESTED** [review]
all good except I want to keep the "iff"

File: notes/05_infomax/1_primer_info_theory.tex (hunk 1 above)
Comment (lcubelongren) [reply notes/05_infomax/1_primer_info_theory.tex]: ok gotcha! didn't know that one, thanks

Comment (lcubelongren) [reply notes/05_infomax/1_primer_info_theory.tex]: and I think it's fine w/o bold, makes sense now

**Pull request merged**